### PR TITLE
Remove CertErrorPage and deps

### PR DESCRIPTION
**Pull request by gildub**
https://github.com/konveyor/virt-ui/issues/227

### CODE REVIEW

**Comment by gildub** [comment]
@mturley, I was thinking of going further and remove some of the code in `fetchHelpers.ts` but since we're still using SSL it's better to just leave as it is for now. What do you think?

**Comment by mturley** [comment]
Well we're still using the self-signed SSL certs on the Node side, but if one of those requests came back with a cert error it wouldn't help to redirect the user to accept it in their browser (Node would still have a problem). I'm fine with removing it, but maybe we can do that in a separate PR to make it easier to revert if I'm wrong :)

File: src/app/queries/fetchHelpers.ts [hunk 1]
```diff
@@ -95,7 +94,6 @@ export const authorizedK8sRequest = async <T>(
     if (isAxiosSelfSignedCertError(error)) {
       const url = `${VIRT_META.clusterApi}/.well-known/oauth-authorization-server`;
       setSelfSignedCertUrl(url);
```
Comment (mturley) [inline src/app/queries/fetchHelpers.ts:96]: Yeah, calling `setSelfSignedCertUrl` does nothing if we don't have the CertErrorPage. We should remove it from the FetchContext as well I think.

**Comment by gildub** [comment]
@mturley, Ok I've kept removal of the use of setSelfsignedCertUrl to a separate PR.

**Review by mturley — APPROVED** [review]

**Pull request merged**